slither/detectors/functions/codex.py

```diff
@@ -52,29 +53,63 @@ def _detect(self) -> List[Output]:
         openai.api_key = api_key
 
         for contract in self.compilation_unit.contracts:
-            prompt = "Is there a vulnerability in this solidity contracts?\n"
+            if self.slither.codex_contracts != "all" and contract.name not in self.slither.codex_contracts.split(","):
+                continue
+            prompt = "Analyze this Solidity contract and find the vulnerabilities. If you find any vulnerabilities, begin the response with {}".format(VULN_FOUND)
```

Comment:
We could probably parameterize the prompt.

Also, maybe we could just always display the Codex response, even if it doesn't find anything.  This would simplify it so we don't need adjust the prompt or look for a special word.